authentication/index.js

```diff
@@ -0,0 +1,2 @@
+module.exports = function(server) {
+};
```

Comment:
For authentication, you'll probably want to go the scheme/strategy [1] route,  and load it as a plugin [2] rather than pass your server object into an exported function

1. http://www.hapijs.com/tutorials/auth
2. http://www.hapijs.com/tutorials/plugins